### PR TITLE
Add status reason to AU eRequesting Diagnostic Request (FHIR-47098)

### PR DESCRIPTION
**Pull request by johnscarter**
This PR fixes [FHIR-47098](https://jira.hl7.org/browse/FHIR-47098).

Added the [Request Status Reason extension](https://hl7.org/fhir/extensions/5.1.0/StructureDefinition-request-statusReason.html) and associated obligations to Au eRequesting Diagnostic Request. The extension is inherited by the Imaging and Pathology Service Requests. [FHIR-47098](https://jira.hl7.org/browse/FHIR-47098)

### CODE REVIEW

File: input/pagecontent/changes.md [hunk 1]
```diff
@@ -14,4 +14,5 @@ This change log documents the significant updates and resolutions implemented fr
   - made ServiceRequest.encounter mandatory (1..1) [FHIR-47008](https://jira.hl7.org/browse/FHIR-47008)
   - added Must Support to ServiceRequest.priority [FHIR-46939](https://jira.hl7.org/browse/FHIR-46939)
   - AU eRequesting ServiceRequest profile renamed to be AU eRequesting Diagnostic Request [FHIR-46842](https://jira.hl7.org/browse/FHIR-46842)
+  - added [Request status reason extension](https://hl7.org/fhir/extensions/StructureDefinition-request-statusReason.html) [FHIR-47098](https://jira.hl7.org/browse/FHIR-47098)
```
Comment (dbojicic) [inline input/pagecontent/changes.md:17]: Suggest changing to something like: 
- added Request status reason extension as a Must Support element with cardinality 0..1

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see comments.

File: input/pagecontent/changes.md (hunk 1 above)
Comment (johnscarter) [reply input/pagecontent/changes.md:17]: done

Comment (dbojicic) [reply input/pagecontent/changes.md]: Thank you, just one minor change to remove the hyperlink from the extension as we don't normally hyperlink from the change log

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see comments.

File: input/pagecontent/changes.md (hunk 1 above)
Comment (JayMurdoch) [reply input/pagecontent/changes.md]: Removed hyperlink on behalf of @johnscarter.

**Review by dbojicic — APPROVED** [review]

**Review by michael-wilson-au — APPROVED** [review]

**Pull request merged**